package/package.json

```diff
@@ -3,8 +3,20 @@
   "version": "0.1.1",
   "type": "module",
   "exports": {
-    "./inline": "./inline.js",
-    "./modal": "./modal.js"
+    "./inline": {
+      "import": "./inline.js",
+      "require": "./inline.cjs"
+    },
+    "./modal": {
+      "import": "./modal.js",
+      "require": "./modal.cjs"
+    }
   },
-  "dependencies": {}
+  "dependencies": {
+    "@forgerock/javascript-sdk": "^3.4.0",
+    "esbuild": "^0.16.6",
```

Comment:
Does this need to be listed in this `/package/package.json`?

Reply:
hmm, probaly not good catch.